CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * [#180](https://github.com/rubocop-hq/rubocop-rails/issues/180): Fix a false positive for `HttpPositionalArguments` when using `get` method with `:to` option. ([@koic][])
 * [#193](https://github.com/rubocop-hq/rubocop-rails/issues/193): Make `Rails/EnvironmentComparison` aware of `Rails.env` is used in RHS or when `!=` is used for comparison. ([@koic][])
+* [#205](https://github.com/rubocop-hq/rubocop-rails/pull/205): Make `Rails/ReversibleMigration` aware of `:to_table` option of `remove_foreign_key`. ([@joshpencheon][])
```

Comment:
Can you fix the following failure test?

```console
Failures:

  1) RuboCop Rails Project changelog has link definitions for all implicit links
     Failure/Error:
       expect(changelog.include?("[#{name}]: http"))
         .to be(true), "CHANGELOG.md is missing a link for #{name}. " \
                       'Please add this link to the bottom of the file.'
     
       CHANGELOG.md is missing a link for @joshpencheon. Please add this link to the bottom of the file.
     # ./spec/project_spec.rb:27:in `block (4 levels) in <top (required)>'
     # ./spec/project_spec.rb:26:in `each'
     # ./spec/project_spec.rb:26:in `block (3 levels) in <top (required)>'

Finished in 1.85 seconds (files took 1.32 seconds to load)
2239 examples, 1 failure
```

https://circleci.com/gh/rubocop-hq/rubocop-rails/2700

You can see also CONTRIBUTING.md:

> If this is your first contribution to RuboCop project, add a link definition for the implicit link to the bottom of the changelog as `[@username]: https://github.com/username`.

https://circleci.com/gh/rubocop-hq/rubocop-rails/2700

Reply:
Done thanks - sorry, the local suite didn't have that failure.